### PR TITLE
[#875] Improve Box component

### LINKED ISSUE #875: Installation: Add Revamped Installation Intro & UI Link Section
- [ ] Add Installation Intro from here: https://docs.google.com/document/d/1Zt5rpGCT5aKG-IpRJwcx81LYPe1uPcikr_5NhtWy_os/edit

- [ ] Build UI Link Section as mocked here:
image
Good implementation on docusaurus of this can be found here: https://vector.dev/docs/setup/installation/package-managers/

### PR DESCRIPTION
**Pull request by AitorAlgorta**
resolves #875

### CODE REVIEW

File: infrastructure/airy.conf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+global:
+  appImageTag: latest
```
Comment (chrismatix) [inline infrastructure/airy.conf:2]: This should not be checked in :)

File: docs/src/components/Box/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const adjust = (color, amount) => {
+  return (
+    '#' +
+    color
+      .replace(/^#/, '')
+      .replace(/../g, color => ('0' + Math.min(255, Math.max(0, parseInt(color, 16) + amount)).toString(16)).substr(-2))
+  );
+};
+
+const Box = ({icon, title, description, link, cutomizedBackgroundColor, cutomizedHoverColor, history}) => {
```
Comment (pascal-airy) [inline docs/src/components/Box/index.js:16]: you are missing an 's' in customized

Comment (AitorAlgorta) [reply docs/src/components/Box/index.js:16]: good catch!

**Review by pascal-airy — APPROVED** [review]
great job!

**Review by lucapette — APPROVED** [review]

File: docs/docs/getting-started/introduction.md [hunk 3]
```diff
@@ -36,6 +39,55 @@ Since Airy's infrastructure is built around Apache Kafka, it can process a large
...
+}}>
+
+<li style={{
+    marginBottom: '12px'
+}}>
+<ButtonBox 
+    icon={() => <DeploySVG />}
+    title='Start Installation' 
+    description='Install Airy with our CLI, locally or in the cloud' 
+    link='getting-started/deployment/vagrant'
```
Comment (pascal-airy) [inline docs/docs/getting-started/introduction.md:58]: This should link to `/cli/installation/`

File: docs/docs/getting-started/introduction.md [hunk 4]
```diff
@@ -36,6 +39,55 @@ Since Airy's infrastructure is built around Apache Kafka, it can process a large
...
+}}>
+
+<li style={{
+    marginBottom: '12px'
+}}>
+<ButtonBox 
+    icon={() => <DiamondSVG />}
+    title='To the Quick Start' 
+    description='Learn the Airy Basics with our Quick Start' 
+    link='getting-started/deployment/production'
```
Comment (pascal-airy) [inline docs/docs/getting-started/introduction.md:78]: and this to `getting-started/quickstart`

**Review by pascal-airy — APPROVED** [review]
🎉

**Pull request merged**